.env.docker

```diff
@@ -0,0 +1,3 @@
+APP_KEY=
+
+DB_DATABASE=docker/database.sqlite
```

Comment:
If using  MariaDB, maybe even with sqlite, this can be removed as it's set in the compose.yml.